internal/anonymize/anonymize_test.go

```diff
@@ -36,7 +36,7 @@ var (
 	commentsQuery            = "INSERT INTO `wp_comments` VALUES (1,1,'A WordPress Commenter','wapuu@wordpress.example','https://wordpress.org/','','2019-06-12 00:59:19','2019-06-12 00:59:19','Hi, this is a comment.\\nTo get started with moderating, editing, and deleting comments, please visit the Comments screen in the dashboard.\\nCommenter avatars come from <a href=\\\"https://gravatar.com\\\">Gravatar</a>.',0,'1','','',0,0);\n"
 	commentsQueryRecompiled  = "insert into wp_comments values (1, 1, 'kamren.ohara', 'michele_barton@example.net', 'http://ebert.com/korey_keeling', '', '2019-06-12 00:59:19', '2019-06-12 00:59:19', 'Hi, this is a comment.\\nTo get started with moderating, editing, and deleting comments, please visit the Comments screen in the dashboard.\\nCommenter avatars come from <a href=\\\"https://gravatar.com\\\">Gravatar</a>.', 0, '1', '', '', 0, 0);\n"
 	usersQuery               = "INSERT INTO `wp_users` VALUES (1,'username','user_pass','username','hosting@humanmade.com','','2019-06-12 00:59:19','',0,'username'),(2,'username','user_pass','username','hosting@humanmade.com','http://notreal.com/username','2019-06-12 00:59:19','',0,'username');\n"
-	usersQueryRecompiled     = "insert into wp_users values (1, 'treva_cremin', '5ccf672d5c73748146be6b37568efa57', 'hailey', 'bernice.heaney@example.net', '', '2019-06-12 00:59:19', '', 0, 'Kylie Rice'), (2, 'eduardo', '266f03a450f9d419543e07208da19c1b', 'albert.okeefe', 'brooke.hayes@example.net', 'http://pfannerstill.net/brando', '2019-06-12 00:59:19', '', 0, 'Ardella Jenkins PhD');\n"
+	usersQueryRecompiled     = "insert into wp_users values (1, 'treva_cremin', '5ccf672d5c73748146be6b37568efa57', 'hailey', 'bernice.heaney@example.net', '', '2019-06-12 00:59:19', '', 0, 'Kylie Rice'), (2, 'eduardo', '266f03a450f9d419543e07208da19c1b', 'albert.okeefe', 'brooke.hayes@example.net', '2_http://pfannerstill.net/brando', '2019-06-12 00:59:19', '', 0, 'Ardella Jenkins PhD');\n"
```

Comment:
Looks like typo. Added `2_` for http url

Reply:
Nope, this is a test for template rendering. I use `"template": "{{printf \"%s\" (index . 0).Val }}_{{fakerInternet.Url}}"

{{printf \"%s\" (index . 0).Val }} uses id of the row